mon-pix/translations/fr.json

```
@@ -566,6 +566,11 @@
                 "subtitle": "Explorez les compétences recommandées pour vous.",
                 "title": "Compétences recommandées"
             },
+            "improvable-competences": {
+                "profile-link": "Voir toutes",
```

```suggestion
                "profile-link": "Toutes les compétences",
```
Comme sur les autres blocs ?

---

Là j'ai purement suivi la maquette, mais ça me paraissait pas assez explicite non plus.